core/controllers/question_editor.py

```diff
@@ -20,9 +20,41 @@
 from core.domain import acl_decorators
 from core.domain import question_domain
 from core.domain import question_services
+from core.domain import skill_domain
+from core.domain import skill_services
 import feconf
 
 
+class QuestionSkillLinkHandler(base.BaseHandler):
+    """A handler for linking and unlinking questions to or from a skill."""
+
+    @acl_decorators.can_manage_question_skill_status
+    def post(self, question_id, skill_id):
+        """Publishes a question by linking it to a skill."""
```

Comment:
I don't think "link a question to a skill" is the same as "publish".

In the future maybe questions can be linked to more than one skill.